src/components/mobile/SettingsPage.js

```diff
@@ -16,6 +17,10 @@ import { Slider } from '../ui/Slider'
 import { Tag, TagList } from '../ui/Tag'
 
 const SettingsPage = () => {
+  const [showDialog, setShowDialog] = useState(false)
+  const open = () => setShowDialog(true)
+  const close = () => setShowDialog(false)
+  const buttonRef = useRef()
```

Comment:
Why is this ref necessary?

Reply:
I implemented the example from Reach UI to get the buttons to open and close the modal and they had refs but it seems to work without so I'll remove em!